libtransmission/quark.cc

```diff
@@ -264,6 +264,7 @@ auto constexpr MyStatic = std::array<std::string_view, TR_N_KEYS>{
     "private"sv,
     "progress"sv,
     "prompt-before-exit"sv,
+    "proxy-url"sv,
```

Comment:
Please change to `proxy_url`.

Reply:
See https://github.com/transmission/transmission/pull/7108

Reply:
Ok, I see. Thank you

Reply:
Yes, thanks for helping to explain.